test/ProtocolGovernance.spec.ts

```diff
@@ -980,9 +1169,35 @@ contract<IProtocolGovernance, CustomContext, DeployOptions>(
             });
 
             describe("access control", () => {
-                xit("allowed: admin", async () => {});
-                xit("denied: deployer", async () => {});
-                xit("denied: random address", async () => {});
+                it("allowed: admin", async () => {
+                    await expect(
+                        this.subject
+                            .connect(this.admin)
+                            .rollbackAllPermissionGrants()
+                    ).to.not.be.reverted;
+                });
+                it("denied: deployer", async () => {
+                    await expect(
+                        this.subject
+                            .connect(this.deployer)
+                            .rollbackAllPermissionGrants()
+                    ).to.be.revertedWith(Exceptions.FORBIDDEN);
+                });
+                pit(
```

Comment:
Let's not use pit in access control, it will give bad spec view. Just do one case with random address